source/develop/release-management/features/hosted-engine-indicate-host-in-web-ui.html.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Hosted Engine indicate hosted engine host in Web UI
+category: feature
+authors: pcbailey
+wiki_category: Feature|Hosted Engine indicate hosted engine host in Web UI
+wiki_title: Features/Hosted Engine indicate hosted engine host in Web UI
+wiki_revision_count: 0
+wiki_last_updated: 2016-04-19
```

Comment:
I just saw this; leave the metadata there for now, and when I review the
blog for grammar and whatnot I can clean the metadata up.

BKP

On Mon, Jul 18, 2016 at 10:37 PM, Phillip Bailey <notifications@github.com>
wrote:

> In
> source/develop/release-management/features/hosted-engine-indicate-host-in-web-ui.html.md
> <https://github.com/oVirt/ovirt-site/pull/187#discussion_r71147720>:
>
> > @@ -0,0 +1,57 @@
> > +---
> > +title: Hosted Engine indicate hosted engine host in Web UI
> > +category: feature
> > +authors: pcbailey
> > +wiki_category: Feature|Hosted Engine indicate hosted engine host in Web UI
> > +wiki_title: Features/Hosted Engine indicate hosted engine host in Web UI
> > +wiki_revision_count: 0
> > +wiki_last_updated: 2016-04-19
>
> @sandrobonazzola <https://github.com/sandrobonazzola> Did you get a
> response from @garrett <https://github.com/garrett> as to whether this
> metadata is necessary or can be removed?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/oVirt/ovirt-site/pull/187/files/11ffa5706efd63187a8e0cc1ee4d815899800e74#r71147720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB6AZckSUnCKdSvuh82IoiId8ePTn972ks5qW4GggaJpZM4ILwo_>
> .
>



-- 
Brian Proffitt
Principal Community Analyst
Open Source and Standards
@TheTechScribe
574.383.9BKP